lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescalesetvm_facts.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_virtualmachinescalesetvm_facts
+version_added: "2.8"
+short_description: Get Azure Virtual Machine Scale Set VM facts.
+description:
+    - Get facts of Azure Virtual Machine Scale Set VMs.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    vmss_name:
+        description:
+            - The name of the VM scale set.
+        required: True
+    instance_id:
+        description:
+            - The instance ID of the virtual machine.
+    tags:
+        description:
+            - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: List VM instances in Virtual Machine ScaleSet
+    azure_rm_computevirtualmachinescalesetvm_facts:
+      resource_group: myResourceGroup
+      vmss_name: myVMSS
+'''
+
+RETURN = '''
+virtualmachines:
+    description: A list of dictionaries containing facts for Virtual Machine Scale Set V M.
+    returned: always
+    type: complex
+    contains:
+        id:
+            description:
+                - Resource Id
+            returned: always
+            type: str
+            sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachineScaleSets/my
+                     VMSS/virtualMachines/2"
+        tags:
+            description:
+                - Resource tags
+            returned: always
+            type: complex
+            sample: { 'tag1': 'abc' }
+        instance_id:
+            description:
+                - Virtual Machine instance Id
+            returned: always
+            type: str
+            sample: 0
+        name:
+            description:
+                - Virtual Machine name
+            returned: always
+            type: str
+            sample: myVMSS_2
+        latest_model_applied:
```

Comment:
the returned info is too limited, what about vnet/subnet, disk, sku, status(power_state), which showed at portal?